veolia-idf-domoticz.py

```diff
@@ -401,77 +540,107 @@ def clean_up(self):
 
         # Remove downloaded file
         try:
-            os.path.exists(self.__full_path_download_file)
-        except:
-            pass
-        else:
-            if os.path.exists(self.__full_path_download_file):
-                self.print("Remove downloaded file "+ self.download_filename, end="") #############################################################
-
+            if not debug and os.path.exists(self.__full_path_download_file):
+                #############################################################
                 # Remove file
-                try:
-                    os.remove(self.__full_path_download_file)
-                except Exception as e:
-                    self.print(str(e),st="EE")
-                else:
-                    self.print(st="ok")
-        pass
+                self.print( "Remove downloaded file " + self.download_filename, end="")
+                os.remove(self.__full_path_download_file)
+            else:
+                self.print(st="ok")
+        except Exception as e:
+            self.print(str(e), st="EE")
 
     def get_file(self):
 
-        self.print('Connexion au site Veolia Eau Ile de France', end="") #############################################################
+        self.print(
+            "Connexion au site Veolia Eau Ile de France", end=""
+        )  #############################################################
         try:
             self.__browser.get(self.__class__.site_url)
         except Exception:
             raise
         else:
             self.print(st="ok")
 
-        self.print('Waiting for Email', end="") #############################################################
+        self.print(
+            "Waiting for Password", end=""
+        )  #############################################################
         try:
-            ep = EC.presence_of_element_located((By.CSS_SELECTOR,"input[inputmode='email'"))
-            el_email = self.__wait.until(ep, message="failed, page timeout (timeout=" + self.configuration['timeout'] + ")")
+            ep = EC.presence_of_element_located(
+                (By.CSS_SELECTOR, 'input[type="password"]')
```

Comment:
Pour ma curiosité, y a une raison d'inverser le Wait for email et le wait for password ?